lib/src/cryptor.dart

```diff
@@ -18,3 +19,33 @@ class Cryptor {
     });
   }
 }
+
+class UniversalCryptor implements Cryptor {
+  Uint8List _iv;
+  Uint8List _key;
+
+  int byte_counter = 0;
+
+  final algorithm = AesCtr.with256bits(
+    macAlgorithm: MacAlgorithm.empty,
+  );
+
+  Future<bool> init(Uint8List key, Uint8List iv) async {
+    _iv = iv;
+    _key = key;
+    byte_counter = 0;
+  }
+
+  Future<Uint8List> crypt(Uint8List data) async {
+    final secretBox = await algorithm.encrypt(
+      data,
+      secretKey: await algorithm.newSecretKeyFromBytes(_key),
```

Comment:
Possibly we could call `algorithm.newSecretKeyFromBytes(_key)` just once in the `init` function.